packages/onchainkit/src/swap/components/SwapSettings.tsx

```diff
@@ -73,29 +72,29 @@ export function SwapSettings({
         {breakpoint === 'sm' ? (
           <>
             {trigger}
-            <FocusTrap active={isOpen}>
-              <DismissableLayer
-                onDismiss={handleClose}
-                triggerRef={triggerRef}
-                preventTriggerEvents={true}
+            <Sheet
+              isOpen={isOpen}
+              onClose={handleClose}
+              side="bottom"
+              title="Swap settings"
+              description="Swap settings"
+            >
+              <div
+                className={cn(
+                  'bg-ock-bg-inverse',
+                  'shadow-ock-default',
+                  'fixed inset-x-0 z-50 transition-[bottom] duration-300 ease-in-out',
```

Comment:
are we getting rid of the `zIndex` const? if not would be nice to use instead of `z-50`